pkg/gui/presentation/commits.go

```diff
@@ -277,6 +277,8 @@ func displayCommit(
 	} else {
 		if len(commit.Tags) > 0 {
 			tagString = theme.DiffTerminalColor.SetBold().Sprint(strings.Join(commit.Tags, " ")) + " "
+		} else if commit.ExtraInfo != "" && commit.ExtraInfo != "(HEAD)" {
```

Comment:
As for the colors: I modeled these after what gets displayed in the expanded commits view, just replacing the actual text with a `*`. So I would find it strange if the colors change when you expand the view.

Considering this, I should probably remove the extra condition for ExtraInfo being "(HEAD)", since the expanded view shows it too. I initially found it redundant because you already have the big bold `<-- YOU ARE HERE ---` display, but being consistent with the expanded view is maybe more important. See 8ec66f5bd342.